README.md

```diff
@@ -27,7 +27,7 @@ yarn add svgson-next
 ## Usage
 
 ```js
-const svgson = require('svgson-next').default
+const svgson = require('svgson-next').parse
```

Comment:
the named export is only for `umd`, [look](https://github.com/elrumordelaluz/svgson-next/blob/master/src/index-es.js#L2) that for `cjs` and `es` is exported as `default`.

Maybe split examples? or so redundant?

Reply:
This is older form to make parser declaration more explicit. Currently I propose using `svgson` with its namespace, so parser and stringifier should be used by `svgson.parse` and `svgson.stringify`. Otherwise it would create naming conflicts in a large code.

Reply:
> Otherwise it would create naming conflicts in a large code.

What kind of namimng conflicts since is `svgson` exported as default no named.
Keep in mind the separation for [umd](https://github.com/elrumordelaluz/svgson-next/blob/master/src/index-umd.js) and [cjs/es](https://github.com/elrumordelaluz/svgson-next/blob/master/src/index-es.js)

Reply:
Think that we are using both `parse` and `stringify` in our code. If we declare the parser as such: 

```
svgson = require('svgson-next').default
```
...then how do we suppose to declare the stringifier? Should we declare a second line? : 

```
svgson = require('svgson-next').default
{stringify} = require('svgson-next')
```

This absolutely looks bad. So we should declare them in one line, but how do we know the `parse` in the first place? We can't without looking into the source code. That was the reason I changed `require('svgson-next').default` to `require('svgson-next').parse`. 

Anyway, let's assume we learnt the name `parse`, so we declared those functions as: 

```
{parse, stringify} = require('svgson-next')
```

Although it seems to work, we probably will have more than one parser/stringifier in our code. That's why it would create conflicts/ambiguities in our code. The solution I end up is using `svgson.` as the namespace. 7 chars more, less confusion. 

> Keep in mind the separation for umd and cjs/es

I'm currently using umd, and I think [cjs/es](https://github.com/elrumordelaluz/svgson-next/blob/master/src/index-es.js) should follow this way. If the above rationale is appropriate for you, we could change cjs/es module accordingly.

Reply:
> then how do we suppose to declare the stringifier

In [tests](https://github.com/elrumordelaluz/svgson-next/blob/master/test.js#L2) shows how could be impored.

The main reason is to default export de core tool of the _library_ and named export the side ones like `stringify` (and possibly others in the future).

To avoid naming collision you could 

```js
import { stringify as myStringifier } from 'svgson'
// or
const { stringify: myStringifier } = require('svgson')
```

let me know what do you think.

Reply:
I still prefer `svgson = require('svgson-next')` notation, but it's up to you. Less confusion is better than less less code.

Reply:
What if we add _named_ export [here](https://github.com/elrumordelaluz/svgson-next/blob/master/src/index-es.js#L3)

```js
export { stringify, svgson as parse } 
```

So who uses `cjs` / `es` could consume the _default_ export or _named_ one, based on preferences, code style, etc.

Reply:
I have no idea on this, I've only limited knowledge on module formats and this is not what I use (I use Browserify). However, it sounds like it will cover both edges.